### PR TITLE
add bs4 library to crypto image

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
Ready

## Related Content Pull Request
[Related PR](https://jira-hq.paloaltonetworks.local/browse/CIAC-2914)

## Description
add bs4 (BeautifulSoup) to crypto image for `MicrosoftGraphMail` to enable HTML stripping easily

### CODE REVIEW

**Review by JudahSchwartz — APPROVED** [review]
please dont merge until content pr is approved

**Comment by GuyAfik** [comment]
merging it as content PR is approved :)

**Pull request merged**
